test/iidm/converter/xml/IdentifiableExtensionXmlSerializerTest.cpp

```diff
@@ -56,7 +56,8 @@ BOOST_AUTO_TEST_CASE(TerminalExtension) {
     Load& load = network.getLoad("LOAD");
     load.addExtension(stdcxx::make_unique<extensions::TerminalMockExt>(load));
 
-    Network network2 = Network::readXml(test::converter::RoundTrip::getVersionedNetwork("eurostag-tutorial-example1-with-terminalMock-ext.xml", IidmXmlVersion::CURRENT_IIDM_XML_VERSION()));
+    std::istringstream stream(test::converter::RoundTrip::getVersionedNetwork("eurostag-tutorial-example1-with-terminalMock-ext.xml", IidmXmlVersion::CURRENT_IIDM_XML_VERSION()));
+    Network network2 = Network::readXml("network.xiidm", stream);
```

Comment:
It's a detail, but you should store the filename in a variable and pass the same value to `getVersionedNetwork` and `readXml` methods

Reply:
Done.